apps/api/src/chapter/repositories/chapter.repository.ts

```diff
@@ -58,16 +61,22 @@ export class ChapterRepository {
       .where(and(eq(chapters.isPublished, true), eq(chapters.id, id)));
   }
 
-  // TODO: check this functions \/
   async getChapterForUser(id: UUIDType, userId: UUIDType) {
-    const [lesson] = await this.db
+    const [chapter] = await this.db
       .select({
+        displayOrder: sql<number>`${lessons.displayOrder}`,
         id: chapters.id,
         title: chapters.title,
         isFreemium: chapters.isFreemium,
         enrolled: sql<boolean>`CASE WHEN ${studentCourses.id} IS NOT NULL THEN true ELSE false END`,
         lessonCount: chapters.lessonCount,
         completedLessonCount: sql<number>`COALESCE(${studentChapterProgress.completedLessonCount}, 0)`,
+        progress: sql<ProgressStatusType>`
```

Comment:
```suggestion
        progress: sql<ProgressStatusTypes>`
```
WDYT about start use types for constants values as name in Pascal case + Types, such as above. As for me there is a little mess with our types and constans